content/en/docs/_index.md

```diff
@@ -62,19 +66,18 @@ Falco is composed of several main components:
 
  - Configuration - defines how Falco is run, what rules to assert, and how to perform alerts. For more information, see [Configuration](configuration).
 
- - Driver - is a software that adheres to the Falco driver specification and sends a stream of system call information. You cannot run Falco without installing a driver.
-Currently, Falco supports the following drivers:
+ - Driver - is a software that adheres to the Falco driver specification and sends a stream of kernel events. Currently, Falco supports the following drivers:
 
-    - (Default) Kernel module built on `libscap` and `libsinsp` C++ libraries
-    - BPF probe built from the same modules
-    - Userspace instrumentation
+    - (Default) Kernel module
+    - Classic BPF probe built
+    - Modern BPF (CO-RE paradigm and more)
 
-    For more information, see [Falco Drivers](/docs/event-sources/drivers/).
+    For more information, see [Falco Drivers](event-sources/drivers).
```

Comment:
Is the website moving to use relative URLs?

Reply:
Not sure about that :sweat_smile: